src/fides/api/ops/models/application_config.py

```diff
@@ -82,3 +112,84 @@ def get_api_set_config(cls, db: Session) -> Dict[str, Any]:
         if config_record:
             return config_record.api_set
         return {}
+
+    @classmethod
+    def get_config_set(cls, db: Session) -> Dict[str, Any]:
+        """
+        Utility method to get the config_set config dict
+
+        An empty `dict` will be returned if no config settings have been set through config
+        """
+        config_record = db.query(cls).first()
+        if config_record:
+            return config_record.config_set
+        return {}
+
+    @classmethod
+    def update_api_set(
+        cls, db: Session, api_set_dict: Dict[str, Any]
+    ) -> ApplicationConfig:
+        """
+        Utility method to set the `api_set` column on the `applicationconfig`
+        db record with the provided dictionary of data.
+
+        Updates are *merged* with any existing `api_set` data in the db.
+        """
+        return cls.create_or_update(db, data={"api_set": api_set_dict})
+
+    @classmethod
+    def update_config_set(cls, db: Session, config: FidesConfig) -> ApplicationConfig:
+        """
+        Utility method to set the `config_set` column on the `applicationconfig`
+        db record by serializing the given `FidesConfig` to a JSON blob.
+        """
+        # need to do this deserialization round-trip to get us a dict
+        # that's serializable as JSON in the db:
+        # pydantic's `.dict()` is NOT serializable as JSON --
+        # see https://github.com/pydantic/pydantic/issues/1409
+        config_dict = loads(config.json())
+        return cls.create_or_update(db, data={"config_set": config_dict})
```

Comment:
note to self this is just called on startup and in tests, and it would be nice if we could do away with this -